diff --git a/src/validate.js b/src/validate.js
index dcf43a702a4b638be6cf3b209931ed7ee4a56dfb..c4fe1f52352c75e0846d26cd3d09b6184e18181c 100644
--- a/src/validate.js
+++ b/src/validate.js
@@ -1,6 +1,6 @@
 import Ajv from "ajv";

-const ajv = new Ajv({ allErrors: true, coerceTypes: true });
+const ajv = new Ajv({ allErrors: true, coerceTypes: true, $data: true });

 const validate = async (parsed, passedRules) => {
   const errors = [];
